modules/monitoring/manifests/checks/mirror.pp

```diff
@@ -8,13 +8,57 @@
 #   Can be used to disable checks/alerts for a given environment.
 #   Default: false
 #
+# [*gcp_mirror_sync_project_id*]
+#   Sets the GCP project id to use for the transfer job.
+#   Default: ''
+#
+# [*gcp_mirror_sync_transfer_job_auth_json*]
+#   Sets the auth JSON required by the Google SDK.
+#   Default: ''
+#
+# [*gcp_mirror_sync_transfer_job_name*]
+#   Sets the transfer job name which controls the mirror sync.
+#   Default: ''
+#
 class monitoring::checks::mirror (
   $enabled = false,
+  $gcp_mirror_sync_project_id = undef,
+  $gcp_mirror_sync_transfer_job_auth_json = undef,
+  $gcp_mirror_sync_transfer_job_name = undef,
 ) {
   icinga::check_config { 'mirror_age':
     source => 'puppet:///modules/monitoring/etc/nagios3/conf.d/check_mirror_age.cfg',
   }
 
+  if ($::aws_migration and $::aws_environment == 'production') {
```

Comment:
Does this conditional serve any purpose? Why not just use `$enabled` and merge it with the block below? Have I missed something?

Reply:
You are correct, moving it to use $enabled